system/lib/emmalloc.c

```diff
@@ -1124,38 +1201,32 @@ void *emmalloc_realloc_uninitialized(void *ptr, size_t size) {
 
 int emmalloc_posix_memalign(void **memptr, size_t alignment, size_t size) {
   assert(memptr);
-  if (alignment % sizeof(void *) != 0) {
-    return EINVAL;
-  }
+  if (alignment % sizeof(void*) != 0) return EINVAL;
+ 
   *memptr = emmalloc_memalign(alignment, size);
-  return *memptr ?  0 : ENOMEM;
+  return *memptr?  0: ENOMEM;
 }
 EMMALLOC_ALIAS(posix_memalign, emmalloc_posix_memalign);
 
 void *emmalloc_calloc(size_t num, size_t size) {
   size_t bytes = num*size;
   void *ptr = emmalloc_memalign(MALLOC_ALIGNMENT, bytes);
-  if (ptr) {
-    memset(ptr, 0, bytes);
-  }
+  if (ptr) memset(ptr, 0, bytes);
   return ptr;
 }
 EMMALLOC_ALIAS(__libc_calloc, emmalloc_calloc);
 EMMALLOC_ALIAS(calloc,        emmalloc_calloc);
 
 static int count_linked_list_size(Region *list) {
   int size = 1;
-  for (Region *i = list->next; i != list; list = list->next) {
-    ++size;
-  }
+  for (Region *i = list->next; i != list; list = list->next) ++ size;
   return size;
 }
 
 static size_t count_linked_list_space(Region *list) {
   size_t space = 0;
-  for (Region *i = list->next; i != list; list = list->next) {
-    space += region_payload_end_ptr(i) - region_payload_start_ptr(i);
-  }
+  for (Region *i = list->next; i != list; list = list->next) space += region_payload_end_ptr(i) - region_payload_start_ptr(i);
```

Comment:
I personally don't like these single line loops, and (I think) its not allowed by our style.